<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="bcgGroup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>Background</string>
          </property>
          <widget class="QPushButton" name="loadBut">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>111</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Load from file</string>
           </property>
          </widget>
          <widget class="QPushButton" name="setBcgBut">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>20</y>
             <width>111</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Set to color</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="pixelGroup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Single Pixel</string>
          </property>
          <widget class="QSpinBox" name="setY">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>20</y>
             <width>81</width>
             <height>21</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="LabX">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
          <widget class="QLabel" name="LabY">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>20</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="setX">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="LabR">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>R:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="setR">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>50</y>
             <width>51</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="setG">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>50</y>
             <width>51</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
          <widget class="QLabel" name="LabG">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>50</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>G:</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="setB">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>50</y>
             <width>51</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
          <widget class="QLabel" name="LabB">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>50</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>B:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>To Be Done</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>IMAGE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
